src/controllers/dynakube/deploymentmetadata/deploymentmetadata.go

```diff
@@ -0,0 +1,51 @@
+package deploymentmetadata
+
+import (
+	"fmt"
+	"strings"
+
+	dynatracev1beta1 "github.com/Dynatrace/dynatrace-operator/src/api/v1beta1"
+	"github.com/Dynatrace/dynatrace-operator/src/version"
+)
+
+type DeploymentMetadata struct {
+	OrchestratorID string
+	DeploymentType string
+}
+
+func GetOneAgentDeploymentType(dynakube dynatracev1beta1.DynaKube) string {
```

Comment:
It should be in `dynakube` package.

Reply:
this kinda contradicts the previous comment
If I do both then I will get circular dependency as the `GetOneAgentDeploymentType` in the `dynakube` package would need to import the `oneagent` package to use the constants, but in the `oneagent` package we already import the `dynakube` package.

I could move both the function and constants to the `dynakube` package